src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -189,6 +189,12 @@ const terminalConfiguration: IConfigurationNode = {
 			default: 4.5,
 			tags: ['accessibility']
 		},
+		[TerminalSettingId.TabSize]: {
+			markdownDescription: localize('terminal.integrated.tabSize', "The number of spaces a tab stop is equal to in the terminal."),
+			type: 'number',
+			minimum: 1,
+			default: 8
```

Comment:
>  I think 4 would be a sensible default, but I've set it to 8 because that's the current tabstop size (it's the default set by xterm).

8 is good as I think some programs could act weird if they depend upon a terminal's typical tab stop width.